dev/Expander/Expander.vcxitems

```diff
@@ -25,10 +25,17 @@
   </ItemGroup>
   <ItemGroup>
     <Page Include="$(MSBuildThisFileDirectory)Expander.xaml">
+      <UseVisualStyle>Latest</UseVisualStyle>
+      <Version>RS1</Version>
+      <Type>DefaultStyle</Type>
+    </Page>
+    <Page Include="$(MSBuildThisFileDirectory)Expander_v2.5.xaml">
```

Comment:
Expander will only have a new style, can you use this as an example for one file that applies to both old and new versions?